.github/workflows/test-suite.yaml

```diff
@@ -1,22 +1,36 @@
 name: Test suite
-
+env:
+  CACHE_NAME: hrajchert-marlowe-ts-sdk
+  ALLOWED_URIS: "https://github.com https://api.github.com"
+  TRUSTED_PUBLIC_KEYS: "cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY= hydra.iohk.io:f/Ea+s+dFdN+3Y/G+FDgSq+a5NEWhJGzdjvKNGv0/EQ= marlowe-temp.cachix.org-1:1gPjVFpu4QjaAT3tRurCioX+BC23V7mjvFwpP5bV0Ec= loony-tools:pr9m4BkM/5/eSTZlkQyRt57Jz7OMBxNSUiMC4FkcNfk="
+  SUBSTITUTERS: "https://cache.nixos.org/ https://cache.iog.io https://marlowe-temp.cachix.org https://cache.zw3rk.com/"
 on:
   pull_request:
     types:
       - opened
       - synchronize
 
 jobs:
-  check-changelog:
+  test-suite:
```

Comment:
I just momentarily forgot we didn't completely disable the tests but only the integration ones...

**Food for thoughts** : We should have a document in the repo, explaining what is the current level testing-wise and some directions how we want to improve it...